silq/parameters/acquisition_parameters.py

```diff
@@ -1858,17 +1860,19 @@ def ESR_frequencies(self, ESR_frequencies: List):
         self.ESR['ESR_pulses'] = updated_ESR_pulses
 
     def analyse(self, traces: Dict[str, Dict[str, np.ndarray]] = None):
-        """Analyse flipping events between nuclear states
+        """Analyse flipping events between nuclear states and determine nuclear state
```

Comment:
This is more in line with what the code is doing.
```suggestion
        """Analyse flipping events between nuclear states and determine the success rate of each ESR pulse (i.e. samples exceeding self.threshold_up_proportion).
```